service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java

```diff
@@ -859,4 +868,294 @@ void changeTariffLocationsStatusParamUnresolvable() {
         assertThrows(BadRequestException.class,
             () -> superAdminService.changeTariffLocationsStatus(1L, dto, "unresolvable"));
     }
+
+    @Test
+    void deactivateTariffByOneRegion() {
+        DetailsOfDeactivateTariffsDto details = ModelUtils.getDetailsOfDeactivateTariffsDtoWithRegion();
+        when(deactivateTariffsForChosenParamRepository.isRegionsExists(anyList())).thenReturn(true);
+        superAdminService.deactivateTariffForChosenParam(details);
+        verify(deactivateTariffsForChosenParamRepository).deactivateTariffsByRegions(List.of(1L));
```

Comment:
please check in every test if there are no more interactions with the mocks

Reply:
checked